src/Validation/Traits/ImageValidationTrait.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace Josegonzalez\Upload\Validation\Traits;
+
+trait ImageValidationTrait
+{
+    /**
+     * Check that the file is above the minimum width requirement
+     *
+     * @param mixed $check Value to check
+     * @param int $width Width of Image
+     * @return bool Success
+     */
+    public static function isAboveMinWidth($check, $width)
+    {
+        // Non-file uploads also mean the height is too big
+        if (!isset($check['tmp_name']) || !strlen($check['tmp_name'])) {
+            return false;
```

Comment:
Is there a way to return true if there wasn't an image in the first place? The 2.0 version had an option to enable these checks but disable them *if* there wasn't an upload.